lib/pages/portal/view/portal_page.dart

```diff
@@ -154,35 +155,28 @@ class _PortalPageState extends State<PortalPage> {
         ),
       );
     } else {
-      final rows = <Row>[];
-
-      for (var i = 0; i < websites.length;) {
-        List<Widget> children = [];
-        for (var j = 0; j < circlesPerRow && i < websites.length; j++, i++) {
-          children.add(websiteCircle(websites[i], circleSize));
-        }
-
-        // Add trailing "plus" button
-        if (i == websites.length) {
-          if (children.length == circlesPerRow) {
-            rows.add(Row(children: children));
-            children = [];
-          }
-          children.add(Container(
-            width: circleSize + 16,
-            child: _AddWebsiteButton(
-              key: ValueKey(
-                  'add_website_${ReCase(category.toLocalizedString()).snakeCase}'),
-              category: category,
-              size: circleSize * 0.6,
-            ),
-          ));
-        }
-
-        rows.add(Row(children: children));
-      }
-
-      content = Column(children: rows);
+      content = Align(
+        alignment: Alignment.centerLeft,
+        child: Wrap(
+          crossAxisAlignment: WrapCrossAlignment.center,
+          children: [
+            ...websites
+                .map(
+                  (website) => websiteCircle(website, maxCircleSize),
+                )
+                .toList(),
+            Container(
+              width: circleSize + 16,
```

Comment:
Do we still need circleSize? Can't we use maxCircleSize everywhere?

Reply:
I was trying to remove it completely, wasn't happy with the result and postponed the task